lib/glific/third_party/sheets/sheets.ex

```diff
@@ -222,4 +225,23 @@ defmodule Glific.Sheets do
         update_sheet_data(sheet_data, attrs)
     end
   end
+
+  @doc """
+  Execute a sheet action
+  """
+  @spec execute(Action.t() | any(), FlowContext.t()) :: {FlowContext.t(), Messages.Message.t()}
+  def execute(action, context) do
+    with {:ok, loaded_sheet} <-
+           Repo.fetch_by(SheetData, %{
+             sheet_id: action.sheet_id,
+             key: action.row,
+             organization_id: context.organization_id
+           }),
+         context <- FlowContext.update_results(context, %{action.result_name => loaded_sheet}) do
```

Comment:
this should be loaded_sheet or loaded_sheet.raw_data ?

Reply:
@pankaj-ag Yes. made requested changes and tested as well